tests/resources/io/simple_notebook.ipynb

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Do we still need simple_notebook?

Reply:
I left io.load_notebook; but removed the "template" argument.  
This is used for testing the direct filepath loading that remains